labs/open-enterprise-agent.md

```diff
@@ -0,0 +1,98 @@
+
+- did:prism
+- did:peer
+- Other did-methods can be verified via a universal resolver
+
+### COMMUNICATION SUPPORT
+
+- DIDCOMM V2
+
+### DIDCOMM PROTOCOLS SUPPORTED
+
+- [Mediator Coordinator](https://didcomm.org/mediator-coordination/2.0/)
+- [DIDComm V2 Messaging](https://identity.foundation/didcomm-messaging/spec)
+- [DIDComm V2 Issue Credential](https://github.com/decentralized-identity/waci-didcomm/tree/main/issue_credential)
+- [DIDComm V2 Present Proof](https://github.com/decentralized-identity/waci-didcomm/blob/main/present_proof/present-proof-v3.md)
+- [DIDComm V2 Report Problem](https://identity.foundation/didcomm-messaging/spec/#problem-reports)
+- [DIDComm V2 Routing Protocol](https://identity.foundation/didcomm-messaging/spec/#routing-protocol-20)
+
+### SUPPORTED ARIES RFC’s
+
+- [0023-did-exchange](https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange)
+- [0434-out-of-band-protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0434-outofband/README.md)
+- [0453-issue-credential-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2)
+- [0453-present-proof-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0454-present-proof-v2)
+
+# Initial Committers
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/FabioPinheiro
+- https://github.com/mineme0110
+- https://github.com/bvoiturier
+- https://github.com/yshyn-iohk
+- https://github.com/CryptoKnightIOG
+- https://github.com/shotexa
+- https://github.com/goncalo-frade-iohk
+- https://github.com/lohanspies
+
+# Initial Maintainers
+
+- https://github.com/bsandmann
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/yshyn-iohk
+
+# Sponsors
+- Lance Byrd - ([lance.byrd@rootsid.com](mailto:lance.byrd@rootsid.com)) - Aries DIDCommV2 WG member
+- Stephen Curran (swcurran@cloudcompass.ca) - ACA-Py Maintainer, Aries RFC Maintainer
+- Timo Glastra - ([timo@animo.id](mailto:timo@animo.id)) - Aries Framework JavaScript Maintainer, ACA-Py Contributor
+- Daniel Hardman - ([daniel.hardman@gmail.com](mailto:daniel.hardman@gmail.com)) - Lead of ToIP Trust Spanning Task Force, Primary editor ToIP ACDC Task Force, Aries RFC Maintainer
+- Darrell O’Donnell - ([darrell.odonnell@continuumloop.com](mailto:darrell.odonnell@continuumloop.com)) - Chair of ToIP Technology Stack WG, Chair of ToIP Technology Architecture Task Force & Lead of ToIP Trust Registry Task Force
+
+# Pre-existing repository
+- https://github.com/input-output-hk/atala-prism-building-blocks
+- For information, the code in this repository makes use of the logback library, which is licensed under GNU Lesser General Public License version 2.1 or Eclipse Public License v1.0 - https://github.com/qos-ch/logback (this repo should not be transfered to the HL Labs organisation)
```

Comment:
I am not sure that I understand this comment. How does the use of the library impact the license of the repo that you are looking to bring over?

Reply:
I fully support this lab but also want to know why this repo-https://github.com/input-output-hk/atala-prism-building-blocks can't be transferred to Hyperledger. Is there any dependency of this repo to code which will be open source in hyperledger-labs.

Reply:
Hey @knagware9   - I think there is a little confusion. The code that we are going to be moving to HL repo is -https://github.com/input-output-hk/atala-prism-building-blocks , the code that we will not transfer is https://github.com/qos-ch/logback.

https://github.com/input-output-hk/atala-prism-building-blocks makes use of [//github.com/qos-ch/logback](https://input-output-rnd.slack.com//github.com/qos-ch/logback) and we were advised by our legal team to include it in the proposal because it is part of the GNU Lesser General Public License (https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License). 

Our legal team has advised that we include https://input-output-rnd.slack.com//github.com/qos-ch/logback from a legal perspective in the proposal but it is a free software license under the GNU Lesser Public License so we do not own it and only make use of it in https://github.com/input-output-hk/atala-prism-building-blocks that we plan to move to the HL repo.

Hope that I was able to answer your question